src/test/java/uk/gov/functional/FunctionalTestBase.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.functional;
+
+import org.skife.jdbi.v2.DBI;
+import org.skife.jdbi.v2.Handle;
+import uk.gov.functional.db.TestEntriesDAO;
+
+public class FunctionalTestBase {
+    protected static String postgresConnectionString = "jdbc:postgresql://localhost:5432/ft_mint";
+    protected static final TestEntriesDAO testEntriesDAO;
+
+    static {
```

Comment:
I spoke to @om-sharma about this. I'm a little uncomfortable with static initializers generally, but @om-sharma made the point that this means we'll create a single DBI object for all tests, rather than one-per-test or one-per-class as we'd get with a `Rule` or `ClassRule`.  I can't think of a better way of achieving this behaviour.